store/dataStore.ts

```diff
@@ -1,14 +1,36 @@
 import { create } from 'zustand';
 import { produce } from 'immer';
-import { IUser } from '@/api/IapiFunctions';
+import {
+  IUser,
+  INFLTeams,
+  IUserWeeklyPick,
+  IWeeklyPicks,
+} from '@/api/IapiFunctions';
 
 interface IDataStoreState {
   user: IUser;
+  NFLTeams: INFLTeams;
+  userWeeklyPick: IUserWeeklyPick;
+  weeklyPicks: IWeeklyPicks;
 }
-
 interface IDataStoreAction {
   updateUser: (id: IUser['id'], email: IUser['email']) => void;
```

Comment:
m: I missed this in Chris's PR, but alphabetize the function names in order.

Reply:
Reordered the functions in alphabetical order.